src/sidebar/services/test/tags-test.js

```diff
@@ -56,6 +56,18 @@ describe('TagsService', () => {
         count: 1,
         updated: stamp,
       },
+      ინგლისური: {
+        // "English" in Georgian
```

Comment:
```suggestion
        // Non-ASCII characters. This is "English" in Georgian
```

Highlight the important aspect of this test case. Most readers will probably guess that the test case matters because of the use of Unicode characters rather than the Georgian language specifically, but I think there is no harm in being explicit.